web/source/osk/visualKeyboard.ts

```diff
@@ -400,10 +400,8 @@ namespace com.keyman.osk {
 
         // Note:  when embedded on Android (as of 14.0, at least), we don't get access to this.
         // Just the base key.
```

Comment:
Is this comment still accurate?

Reply:
I think the comment is outdated now, after the gesture rework, but not in the way you'd expect.

Turns out, it's just not available directly at all now, hence the need for the newly-added parameter to the function.  Of course, that means it's no longer special to Android, so I'll rip the comment out.